aten/src/ATen/Declarations.yaml

```diff
@@ -22710,7 +22710,7 @@
   abstract: true
   auto_gpu: true
   with_gil: false
-- name: _cast_uint8_t
+- name: _cast_Byte
```

Comment:
this commit looks sketchy as well. cc: @gchanan 
I'm not sure this change should've gone into this PR